src/.classpath

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think you shouldn't modify the class path, since we're all working on the same projects this element is unnecessary and may create merge conflict/duplicate code